lib/components/Alert/Alert.test.tsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
 import { Alert } from '../';
+import { AlertProps } from './Alert';
 import { render } from 'react-testing-library';
 import forEachTheme from '../../test/utils/forEachTheme';
 
 describe('Alert', () => {
   forEachTheme(({ ThemeProvider }) => {
     describe('Tones', () => {
-      const tones = ['critical', 'info'];
+      const tones: Array<AlertProps['tone']> = ['critical', 'info', undefined];
 
       tones.forEach(tone => {
-        test(tone, () => {
+        test(tone || 'default', () => {
```

Comment:
Is it equivalent to push 'default' into the `tones` array above in place of `undefined`?

Reply:
`undefined` is one of the prop values, since it's optional. The `default` string here is just so that the test has a name.

TypeScript actually caught this one for me, telling me that `test` accepts a string as the first arg and I was unwittingly passing a value that could be undefined 😄